README.md

```diff
@@ -24,4 +24,6 @@ _NB: be	sure to	use the	`--destination`	flag if	you're not in a	working	drupal e
 
 ## Installation
 
-`composer create-project --no-dev -d ~/.terminus/plugins aaronbauman/terminus-config-export`
```

Comment:
Maybe just keep both lines for now?
e.g. 

terminus v2: 
`composer create-project`

terminus v3:
`terminus self:plugin:install...`

Reply:
Great idea! :+1: